### PR TITLE
Split content from `FilterModal` into `FilterContent` 

### PR DESCRIPTION
**Pull request by oisincoveney**

### CODE REVIEW

File: frontend/src/metabase/querying/components/FilterContent/FilterBody.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  groupItems,
+  onChange,
+  onInput,
+  onTabChange,
+  query,
+  searching,
+  tab,
+  version,
+}: FilterBodyProps) => (
+  <>
```
Comment (heypoom) [inline frontend/src/metabase/querying/components/FilterContent/FilterBody.tsx:30]: nit: you might not need the outer fragment here as both of them only has one element?

**Review by heypoom — APPROVED** [review]
Looks good to me!

File: frontend/src/metabase/querying/components/FilterContent/FilterBody.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/querying/components/FilterContent/FilterBody.tsx:30]: yeah you're right

**Review by WiNloSt — COMMENTED** [review]
The change itself looks good. But I lack the context why these abstractions are needed. e.g. If you say these will be reused in x components then it makes sense to have `FilterContent`

**Comment by oisincoveney** [comment]
> The change itself looks good. But I lack the context why these abstractions are needed. e.g. If you say these will be reused in x components then it makes sense to have `FilterContent`

The context is that I want to use these components in the SDK - right now the content components are coupled with the modal itself, which doesn't really help when we want to give clients some control over when and where the filter panel would appear. I think it'll make more sense in the next PR when I actually expose the filter panel in the SDK

**Pull request merged**